### DISCUSSION TITLE
posts/CSSdigain/

### DISCUSSION THREAD

**Discussion by giscus** [sunhew/sunhew.github.io#8] [Announcements]
# posts/CSSdigain/

CSS를 활용한 디자인

https://sunhew.github.io/posts/CSSdigain/

**Comment by KIMJW04** [comment]
저희 할머니도 이해할 수 있는 글이네요

**Comment by Kingsong97** [comment]
저희 할아버지가 이거 보고 따라하셨어요! 가독성 굿

**Comment by hyunminkim93** [comment]
저희 외할머니가 보고 따라하셨어요! 좋네요 ㅎㅎ

**Comment by gnlgk** [comment]
정리 잘하셨네요~
